services/api/blocksim_ratelimiter.go

```diff
@@ -50,6 +51,33 @@ func NewBlockSimulationRateLimiter(blockSimURL string) *BlockSimulationRateLimit
 	}
 }
 
+func (b *BlockSimulationRateLimiter) TobSim(context context.Context, tobValidationRequest *common.TobValidationRequest) (requestErr, validationErr error) {
```

Comment:
nit: this looks a lot like `Send` can we make a more generic send and pass in the path ?

Reply:
yeah thats a good idea. will refactor the code.